sorc/app_build.sh

```diff
@@ -25,6 +25,8 @@ OPTIONS
   --disable-options="OPTION1,OPTION2,..."
       disable ufs-weather-model options; delimited with ','
       (e.g. 32BIT | INLINE_POST | UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
+  --extrn
+      check out external components
```

Comment:
Here as well.

Reply:
I've made all these changes to app_build.sh on Hera and successfully pushed everything back to AnnaSmoot-NOAA:build_updates! How do I update the code in this PR?